contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -31,6 +32,11 @@ import "../utils/zeppelin/SafeMath.sol";
  *      2. Network Fee distribtuion. per epoch Kyber fee Handler reads current distribution from Kyber DAO.
```

Comment:
pls help to fix this typo: "distribtuion"

Reply:
fixed